### ISSUE TITLE
Traffic rendering issue

### ISSUE THREAD

**Issue by bojark** [klyte45/DynamicTextProps#26]
The mod seems to break in-game traffic rendering, see this vid: https://youtu.be/ENDvcuTukkw

The issue appeared in last 10 days as I lunced game on April 18 and everything was OK.

**Comment by klyte45** [comment]
Transferring to the actual mod repository
Well, it seems to be an issue related to an exception being thrown. I need your output log to check it better. See instructions at "Report-a-bug helper" button in the mod options in the pause/options menu, if you don't know how to do it

**Comment by bojark** [comment]
I've unsubscribed for the mod and played a game without it for a while, so my curent output log is clear of errors.

But when I subscribed to the mod again and loaded my old save (the one I made BEFORE unsubscribing) the problem was gone and now it works fine.

**Comment by bojark** [comment]
thank you for your work, Leandro

**Comment by bojark** [comment]
Nope, the problem is right here.
It appeared after I loaded the game saved after the unsubscribing. 
Here is the log.
[output_log.txt](https://github.com/klyte45/WriteTheSigns/files/6409242/output_log.txt)

**Comment by klyte45** [comment]
I did a possible fix in the current version at [alpha branch.](https://github.com/klyte45/WriteTheSigns/tree/alpha/_requiredDLLs) 
Please give a try with this version to check if it was solved

**Comment by bojark** [comment]
Checked alpha-branch version (put it in "Mods" folder manualy, turned in in game content manager), loaded the last game and the problem appeared.

note that I never deleted content on \AppData\Local\Colossal Order\Cities_Skylines\Klyte45Mods\WriteTheSigns when unsubscribing or resubscribing the mod. Can it be the issue? may be the new version conflicts with the stuff I made in the older one?

[output_log 020521.txt](https://github.com/klyte45/WriteTheSigns/files/6411377/output_log.020521.txt)
